### ISSUE TITLE
Fee slider: hardcoded minimum is too high.

### ISSUE THREAD

**Issue by schildbach** [spesmilo/electrum#2614]
Currently, the minimum is hardcoded to 0.3 mBTC/kb. At the current transaction volume, it makes sense to slide it as low as 0.05 mBTC/kb (if you dare to wait a bit).

**Comment by ecdsa** [comment]
ack. I think we need a non-linear scale.

**Comment by Engelberg** [comment]
Yesterday, when I started up Electrum the lowest value on the non-dynamic fee slider was 1 satoshi per byte, and today (not having restarted Electrum), the minimum value shown on the slider is 5 satoshis/byte.  What is affecting the minimum value of the slider?

**Comment by ecdsa** [comment]
maybe you modified the fee rate with setconfig ?

**Comment by Engelberg** [comment]
I didn't, but I think when I installed Electrum 3.0.2 it retained some slider settings from the previous installation, and then the first time I moved the slider, it started using some newer scheme which sets the lowest notch on the slider to 5 sat/byte.  That's my best guess.

I am seeing a seemingly related glitch in the replace-by-fee dialog, where it shows the lowest notch on the slider as 5 sat/byte, but the calculation is actually using some other number like 2 sat/byte.  I have to drag the slider away and back to the lowest position in order for the computation to truly reflect the 5 sat/byte setting.
